src/platforms/common/data-management/debug-files/source-context.mdx

```diff
@@ -58,6 +58,12 @@ For Flutter Android, iOS, and macOS, follow the instructions in the [Debug Symbo
 
 </PlatformSection>
 
+<PlatformSection supported={["apple"]}>
+
+To get source context for your Apple project, follow the instructions in the [Uploading Debug Symbols](/platforms/apple/dsym/#pploading-source-context) guide.
```

Comment:
```suggestion
To get source context for your Apple project, follow the instructions in the [Uploading Debug Symbols](/platforms/apple/sourcecontext) guide.
```

Since source context has its own page now, we can link it there.